docs/introduction.org

```diff
@@ -0,0 +1,106 @@
+* Introduction
+
+Nowadays, multiple languages have emerged to compete with the well-established C language.
+In fact, C is widely used in many places, learned by a lot of engineers, and used for many purposes.
+C is a powerful tool, /but with great power comes great responsibility/, in fact, memory leaks can easily occur by mistake.
+
+That's why new languages come to find a better way to handle errors and memory.
+The Rust language relies strongly on its borrow checker to avoid safe memory but it comes at a cost: increased code complexity
```

Comment:
"to avoid safe memory" would rather be "to ensure safe memory" or "to avoid memory issues"